### PR TITLE
TDRD-318-Metadata-validation-properties-user-friendly-error-message-file

### PR DESCRIPTION
**Pull request by ian-hoyle**

### CODE REVIEW

**Comment by TomJKing** [comment]
@ian-hoyle Do you want feed back on this?

**Comment by ian-hoyle** [comment]
> @ian-hoyle Do you want feed back on this?

Yes please

File: validation-messages/validation-messages.properties [hunk 1]
```diff
@@ -0,0 +1,45 @@
+SCHEMA_CLOSURE_CLOSED.foi_exemption_asserted.type=Must be provided for a closed record
+SCHEMA_CLOSURE_CLOSED.foi_exemption_code.type=Must be provided for a closed record
+SCHEMA_CLOSURE_CLOSED.closure_period.type=Must be provided for a closed record
+SCHEMA_CLOSURE_CLOSED.closure_period.type=Must be provided for a closed record
```
Comment (vimleshtna) [inline validation-messages/validation-messages.properties:4]: Duplicate property name and value (line no. 3 and 4)

File: validation-messages/validation-messages.properties [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+SCHEMA_CLOSURE_CLOSED.closure_start_date.type=Must be provided for a closed record
+SCHEMA_CLOSURE_CLOSED.title_closed.type=Must be provided for a closed record
+SCHEMA_CLOSURE_CLOSED.description_closed.type=Must be provided for a closed record
+SCHEMA_CLOSURE_OPEN.foi_exemption_asserted.type=Must be empty for an open record
+SCHEMA_CLOSURE_OPEN.foi_exemption_code.type=Must be empty for an open record
+SCHEMA_CLOSURE_OPEN.closure_period.type=Must be empty for an open record
+SCHEMA_CLOSURE_OPEN.closure_period.type=Must be empty for an open record
```
Comment (vimleshtna) [inline validation-messages/validation-messages.properties:11]: Duplicate property name and value (line no. 10 and 11)

File: validation-messages/validation-messages.properties [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+SCHEMA_BASE.file_name_translation.type=Must be a text field with a maximum length of 8000 characters
+SCHEMA_BASE.file_name_translation.maxLength="This is more than 8000 characters, this field has a maximum length of 8000 characters"
+SCHEMA_BASE.file_name.minLength=Must be at least 1 character
+SCHEMA_BASE.language.type="Language must be capitalised English or Welsh, if a record is in both languages you must use a pipe separated list, ie. English|Welsh"
+SCHEMA_BASE.description_alternate.type=Must be a text field with a maximum length of 8000 characters
+SCHEMA_BASE.description_alternate.maxLength="More than 8000 characters, this field has a maximum length of 8000 characters"
+SCHEMA_BASE.description_closed.type=Must be Yes or No
+SCHEMA_BASE.foi_exemption_asserted.daBeforeToday=Must be a date in the format yyyy-mm-dd
+SCHEMA_BASE.foi_exemption_asserted.format.date="We are unable to recognise this as a valid date format, please provide a date in the format yyyy-mm-dd"
+SCHEMA_BASE.foi_exemption_asserted.daBeforeToday="This date is in the future, please correct."
```
Comment (vimleshtna) [inline validation-messages/validation-messages.properties:33]: Duplicate property name (line no. 31 and 33)

**Review by vimleshtna — COMMENTED** [review]
Looks good. Just a few minor comments. Thanks

**Review by vimleshtna — APPROVED** [review]
Looks good. Thanks

**Pull request merged**